n2y/plugins/jinjarenderpage.py

```diff
@@ -113,7 +113,7 @@ def join_to(foreign_keys, table, primary_key='id'):
 
 def list_matches(string, text):
     return list(re.finditer(
-        '(?<![a-zA-Z])' + _canonicalize(string) + '(?![a-zA-Z])',
+        '(?<![a-zA-Z])' + re.escape(_canonicalize(string)) + '(?![a-zA-Z])',
```

Comment:
👍 nice! Where are the unit tests you added? We should probably commit those too.